src/components/TutorDashboard/index.js

```diff
@@ -405,24 +407,29 @@ function Tutor(props) {
         <Grid container className={classes.filters} mb={2}>
           <Grid item>
             <Button
-              variant="contained"
+              // "contained"
               className={classes.python}
               onClick={() => {
                 setVolunteer(filterPathway(1, cacheVolunteer));
                 setSelectedPathway("Python");
+                setPathwayClicked(true);
+                setPathwayTwoClicked(false);
               }}
+              variant={pathwayClicked ? "contained" : "outlined"}
             >
               Python ({pathwayCount?.python})
             </Button>
           </Grid>
           <Grid item>
             <Button
-              variant="outlined"
               className={classes.learningTrack2}
               onClick={() => {
                 setVolunteer(filterPathway(2, cacheVolunteer));
                 setSelectedPathway("Spoken English");
+                setPathwayClicked(false);
+                setPathwayTwoClicked(true);
               }}
+              variant={pathwayTwoClicked ? "contained" : "outlined"}
```

Comment:
Instead of how you wrote this, you can remove `pathwayTwoClicked` and just write the following:
```suggestion
              variant={pathwayClicked ? "outlined" : "contained"}
```